server/src/test/java/org/elasticsearch/search/fetch/subphase/FieldFetcherTests.java

```diff
@@ -394,6 +394,165 @@ public void testArrayValueMappers() throws IOException {
         assertThat(field.getValues().size(), equalTo(2));
     }
 
+    public void testGeopointArrayInObject() throws IOException {
+        MapperService mapperService = createMapperService();
+        {
+            XContentBuilder source = XContentFactory.jsonBuilder()
```

Comment:
There's a `fetchFields` method that takes a String rather than an XContentBuilder, so I think it's worth changing these to use multiline strings?  They're just so much easier to read.